ExtendedDescription.md

```diff
@@ -0,0 +1,130 @@
+## About nelson.social
+
+Welcome to nelson.social, an online space dedicated to the community of Nelson, BC! We’re excited to build a place where people from all walks of life can come together to share ideas and passions, build connections, or just hang out and chat.
+
+Here at nelson.social, we believe in the power of community and the importance of supporting one another in order to enrich our community. Whether you are a lifelong resident of Nelson, new to the region, or just passing through, we invite you to join the conversation and be a part of our community.
+
+In this space, you can share your thoughts and ideas, ask questions of your neighbours, and even plan events and activities. You can also connect with people from all over the world, because nelson.social is part of the [Mastodon](https://github.com/mastodon/) network. We like to think of it like a big WhatsApp or iMessage group, but where you can choose who to share with and whose messages you see, even if they aren’t members of nelson.social.
+
+We encourage you to be respectful and considerate of others, and to use this space to help build a stronger, more connected community in Nelson. We have a simple set of rules for the space that help the team that runs nelson.social ensure that we can be welcoming for everyone. We encourage you to read them, and also reach out to us if you encounter any problems, or have any questions.
```

Comment:
We use "encourage" twice here, can we use a stronger word?

Expect, demand, require?

Reply:
Sure, well spotted. I think this was copy @blaine used ChatGPT to generate!

Reply:
```suggestion
We expect our members to be respectful and considerate of others, and to use this space to help build a stronger, more connected community in Nelson. We have a simple set of rules for the space that help the team that runs nelson.social ensure that we can be welcoming for everyone. Please read them, and reach out to us if you encounter any problems, or have any questions.
```